### ISSUE TITLE
Rendering disjunctive lists with brackets

### ISSUE THREAD

**Issue by aewingate** [IUBLibTech/newton_chymistry#126]
Going over encoding guidelines for update to P5, and I found that we say for disjunctive lists that there should be a pop-up saying that the manuscript has brackets here. Newton-dev does not do this, and I think it would actually be preferable (if possible) to surround the list with brackets to reflect how the manuscript looks better.

For example, here's ALCH00014:
![image](https://github.com/IUBLibTech/newton_chymistry/assets/72820848/4ff9fa7a-e63b-4299-b7c1-9631cd16e139)

And the XML:
```xml
<list type="disjunctive" rend="bracket"><item>deferring</item><item>suspension</item></list>
```
It would be great if we could put curly brackets on either side of the disjunctive list.

**Comment by tubesoft** [comment]
I would be happy to implement the feature, but what about adding { and } before and after?
Although the curly brackets cannot be the size of the two lines, is it a really serious matter?
If it matters and if there are more similar cases that can be applied, I would try to add some code to render a larger size bracket.

**Comment by aewingate** [comment]
I agree that adding the { and } before and after is a better choice than what's described in the encoding guide right now. And I don't even think that it's super necessary to make them bigger. Let me check with Bill and Jim today about it.
